src/Movie/Movie.js

```diff
@@ -3,7 +3,7 @@ import './Movie.css'
 
 const Movie = ({id, title, releaseDate, averageRating, backdropPath, posterPath}) => {
   return (
-    <section className='Movie' style={{ backgroundImage: `url(${backdropPath})`}} id={id}>
+    <section className='Movie' aria-label='movie' style={{ backgroundImage: `url(${backdropPath})`}} id={id}>
```

Comment:
You should try and keep labels semantic and specific when possible. If it weren't for the test, would you want there to  be a label here that says "movie"?